### DISCUSSION TITLE
Confirmed Working: Monterey 12.2.1 TP-Link Nano AC600 SIP Re-Enabled

### DISCUSSION THREAD

**Discussion by deurbroucq** [chris1111/Wireless-USB-OC-Big-Sur-Adapter#207] [General]
I can confirm that it is working on Monterey 12.2.1 with SIP re-enabled after installation. I am using a TP-Link Nano AC600 adapter. Thank you!

**Comment by chris1111** [comment]
Yes [Confirmed](https://github.com/chris1111/Wireless-USB-OC-Big-Sur-Adapter/issues/208)
